### ISSUE TITLE
No https support?

### ISSUE THREAD

**Issue by parrker** [TimPietrusky/KudosPlease#5]
It looks like the plugin does not on pages served via https, because browsers block ajax request to http://api.kudosplease.com/"+q 

Is there a way to fix it for my users?

**Comment by fredrikaverpil** [comment]
+1 for https support. I use KudosPlease on my github-hosted jekyll site. Right now the button works with http, but not with https:

Works: http://fredrikaverpil.github.io/2015/12/28/python-web-server/
Does not work: https://fredrikaverpil.github.io/2015/12/28/python-web-server/

**Comment by afollestad** [comment]
+1

**Comment by nielsole** [comment]
Hi,
I wrote a clone in Golang. There is a host at https://kudos.niels-ole.com/ (also with http support). 
You can simply replace the references to the server in the JS with that address. 
I wrote two lines at https://www.niels-ole.com/kudos/blog/2016/11/04/now-with-kudos.html 
Will release the code shortly.

**Comment by RCasatta** [comment]
+1
using niels-ole https endpoint now. Cannot import old kudos :(

**Comment by MiteshShah** [comment]
@nielsole Thanks for HTTPS support
Also can you implements total website kudos

```
https://kudos.niels-ole.com/?url=https://miteshshah.github.io/windows/how-to-change-windows-7-product-key/
22
```
```
https://kudos.niels-ole.com/?url=https://miteshshah.github.io/
0
```

Ideally we should return all the kudos for the same domain.

**Comment by nielsole** [comment]
Hi,

the feature is now live. e.g.: https://kudos.niels-ole.com/?domain=miteshshah.github.io
It would be best if further issues and discussions would take place at https://github.com/nielsole/kudos

**Comment by MiteshShah** [comment]
@nielsole Sure Not aware of that repo https://github.com/nielsole/kudos
Thanks for the quick implements :)

**Comment by Kishoreramesh07** [comment]
@nielsole HTTPS expired can you renew it

**Comment by nielsole** [comment]
yeah, migrating servers right now, will fix it probably today or tomorrow

**Comment by Kishoreramesh07** [comment]
Thanks for the quick fix
